resources/metadata/xsDeploy.yaml

```diff
@@ -0,0 +1,128 @@
+metadata:
+  name: xsDeploy
+  description: Performs xs deployment
+  longDescription: |
+    Performs xs deployment
+
+spec:
+  inputs:
+    resources:
```

Comment:
Jenkins credentials are missing here ...

Reply:
Fixed in the meantime